xonsh/tools.py

```diff
@@ -555,6 +555,21 @@ def ensure_int_or_slice(x):
         return int(x)
 
 
+def is_completions_display_value(x):
+    return x in {'none', 'single', 'multi'}
+
+
+def to_completions_display_value(x):
+    if str(x).lower() in {'none', 'false'}:
+        return 'none'
+    if str(x).lower() in {'multi', 'true'}:
+        return 'multi'
+    if str(x).lower() in {'single'}:
```

Comment:
> Do you mean this as in x = str(x).lower() ; if x in {...?

yes.

> Why would this be better than my current version?

I find it easier to read and follow. Just a stylistic thing. Not a deal breaker for this PR. Just something I would appreciate.